dashboard-for-desktop/articles/dashboard-designer/providing-data/binding-to-federated-data-source.md

```diff
@@ -25,27 +43,29 @@ To create a federated data source with the Data Source Wizard, perform the follo
 
    **Join**
 
-   Combines rows from two or more tables based on a shared column. The 'join' type specifies records with matching values in both tables.
-     
-    Drag-and-drop the required data sources, specify the related column (to create a relationship between tables), and select columns to include in the query. Columns included in the query are displayed in the bottom pane - where you can configure their settings.
+   Drag-and-drop the data sources, specify the related column to create the relationship between tables, and select the columns you want to include in the query. Use _* (All Columns)_ to get a complete list of the columns from a table. Columns included in the query are displayed in the bottom pane where you can configure their settings.
 
    ![Federated_data_source_settings](../../../images/federated-source-settings.png)
 
    **Union**
 
-   Combines rows from two or more tables into a single data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain similar columns with the same name. Data types for these columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).
-
-   Drag-and-drop the data sources to combine them into a single data source. Unlike the join query, you cannot specify columns to include in the union query. Columns included in the query are displayed in the bottom pane, where you can specify the aliases of the newly created columns.
+   Double-click the data sources you want to combine. Unlike Join queries, you cannot select individual columns. Columns included in the query are displayed in the bottom pane where you can specify their aliases.
```

Comment:
You capitalized "Join" here, but on line 15 it's not capitalized. Which one is correct?

Reply:
```suggestion
   Double-click the data sources you want to combine. Unlike join queries, you cannot select individual columns. Columns included in the query are displayed in the bottom pane where you can specify their aliases.
```